docs/prover/cli/options.md

```diff
@@ -646,39 +650,28 @@ Upon instruction from the Certora team.
 
 **Example**
 
-`--javaArgs '"-Dcvt.default.parallelism=2"'` - will set the number of “tasks” that can run in parallel to 2.
-
-### `--rerun_verification`
-
-**What does it do?**  
-Repeats a previous run, but skips CVL compilation and TAC optimization phases for a single rule, by using a saved binary file from a previous run.
-
-**When to use it?**  
-When you want to run the same configuration, but save some run-time (for example when encountering a timeout). This should be used with the same parameters to the solver (i.e. same source, specs and optimization configurations). To save a binary to rerun use `--settings -saveRerunData`, and the binary file will be save in outputs.  
-**Example**
-
-`certoraRun Bank.sol --verify Bank:bank.spec --settings -saveRerunData`
-
-`certoraRun Bank.sol --verify Bank:bank.spec --rerun_verification rerun_checkBank.rerunbin`
+`--java_args '"-Dcvt.default.parallelism=2"'` - will set the number of “tasks” that can run in parallel to 2.
 
-(--settings)=
-### `--settings`
+(--prover_args)=
+### `--prover_args`
```

Comment:
this is the header you changed, and more importantly notice that in line 655 you changed the string used to reference this header from other files.